jobs/build/ocp4/build.groovy

```diff
@@ -476,14 +476,16 @@ def stageBuildImages() {
         def archReleaseStates = commonlib.ocp4ReleaseState[version.stream]
         // If any arch is GA, use signed for everything. See stageBuildCompose for details.
         def signing_mode = archReleaseStates['release']?'signed':'unsigned'
-
+        // TODO: make this qe additional registry parameterized?
+        buildlib.registry_quay_qe_login()
         def cmd =
             """
             ${doozerOpts}
             ${includeExclude "images", buildPlan.imagesIncluded, buildPlan.imagesExcluded}
             images:build
             --repo-type ${signing_mode}
             --push-to-defaults
+            --registry-config=./qe_quay_config
```

Comment:
you'll also need `--filter-by-os='.*'` in order for it to sync the manifest list.